packages/widget-chat/src/containers/message-composer/index.test.js

```diff
@@ -1,20 +1,22 @@
 import React from 'react';
-import {findRenderedDOMComponentWithClass, renderIntoDocument} from 'react-addons-test-utils';
+import renderer from 'react-test-renderer';
 
 import MessageComposer from '.';
 
-let component;
-
 describe(`MessageComposer component`, () => {
-  beforeEach(() => {
-    component = renderIntoDocument(
-      <MessageComposer />
-    );
-  });
+  const component = renderer.create(
+    <MessageComposer
+      placeholder="Message Placeholder"
+      value="This is a message"
+    />
+  );
 
-  it(`renders`, () => {
-    const renderedComponent = findRenderedDOMComponentWithClass(component, `message-composer`);
-    expect(renderedComponent).toBeDefined();
+  it(`renders properly`, () => {
+    expect(component).toMatchSnapshot();
   });
 
+  // it(`sends message properly`, () => {});
```

Comment:
yea, i added some other to do code as well. Just things we need to figure out how to test before we can call the components complete.